spec/models/vacation_spec.rb

```diff
@@ -72,6 +72,16 @@
 
       it { expect(vacation).to_not be_valid }
     end
+
+    context 'when the duration of the vacation is less than 10 days' do
```

Comment:
and when vacation duration is equal 10 days and more than 10 days?

Reply:
Done